packages/web-components/src/components/ic-date-input/ic-date-input.stories.mdx

```diff
@@ -269,12 +328,42 @@ import readme from "./readme.md";
   <Story name="Dates from now disabled" parameters={{ loki: { skip: true } }}>
     {html`<ic-date-input
         label="When would you like to collect your coffee?"
-        disabled-dates="from-now"
+        disable-from-now
       ></ic-date-input>
       <ic-typography>Enter a date in the future</ic-typography> `}
   </Story>
 </Canvas>
 
+### Disabled days
+
+<Canvas>
+  <Story name="Disabled days" parameters={{ loki: { skip: true } }}>
+    {html`<ic-date-input
+        label="When would you like to collect your coffee?"
+      ></ic-date-input>
+      <script>
+        var dateInput = document.querySelector("ic-date-input");
+        dateInput.disableDays = [0, 1];
+      </script> `}
+  </Story>
+</Canvas>
+
+### Min and max
+
+<Canvas>
+  <Story name="Min and max" parameters={{ loki: { skip: true } }}>
+    {html`<ic-date-input
+        label="When would you like to collect your coffee?"
+        min="2023-01-02"
+        max="2023-07-20"
```

Comment:
Should max/min also accept the same date formats as `.value` to keep things consistent?

Reply:
I thought it did as it accepts Date object or string

Reply:
If I pass through a max date as '20-07-2023', 'Invalid Date' is returned from `this.maxDate' within 'watchMaxHandler'.